Options.lua

```diff
@@ -162,12 +162,28 @@ function Opt:Show()
         end)
     minimap:SetPoint("TOPLEFT", purge, "BOTTOMLEFT", 0, -8)
 
+    local disableDungeons = checkbox("Disable in dungeons (save CPU)",
+        "With this is checked, mob kills in dungeons won't be counted.",
```

Comment:
With -> When

Reply:
Well, English is not my mother tongue... All existing options use "with" :-D
But no problem, I'll commit a fix :-)